src/main/resources/com/puppycrawl/tools/checkstyle/meta/checks/modifier/RedundantModifierCheck.xml

```diff
@@ -116,6 +120,9 @@
  }
  &lt;/pre&gt;</description>
          <properties>
+            <property default-value="17" name="jdkVersion" type="int">
+               <description>Allow the check to be aware of the Java version.</description>
```

Comment:
```suggestion
               <description>Set the Java version that you are using.</description>
```

Reply:
We aren't allowing anything, this property is always set now

Reply:
done